package.json

```diff
@@ -22,7 +22,9 @@
     "url": "git@github.com:cisco-ce/jsxapi.git"
   },
   "scripts": {
-    "build": "npm run build:js",
+    "build": "run-s build:js build:**",
```

Comment:
Does `build:**` run `build:js` again?

Reply:
Nope, which is why I did it like this, to run `tsc` first.

Reply:
I don't know `npm-run-all` though, I just wanted to avoid retyping. I'll look into reducing the `--no-minify` duplication though.